Sources/MusicXML/Complex Types/Appearance.swift

```diff
@@ -10,17 +10,31 @@
 /// and standard distances between notation elements, plus an extension element for other aspects of
 /// appearance.
 public struct Appearance {
-    public let lineWidth: LineWidth?
-    public let noteSize: NoteSizeType?
-    public let distance: Distance?
-    // TODO: other-appearance?
+    
+    public let lineWidths: [LineWidth]
+    public let noteSizes: [NoteSize]
+    public let distances: [Distance]
+    public let otherAppearances: [OtherAppearance]
 
-    public init(lineWidth: LineWidth? = nil, noteSize: NoteSizeType? = nil, distance: Distance? = nil) {
-        self.lineWidth = lineWidth
-        self.noteSize = noteSize
-        self.distance = distance
+    public init(
+        lineWidths: [LineWidth] = [],
```

Comment:
Should we standardize `0...*` elements's initializer to be defaulted to empty array in the whole codebase? That would save a lot of cliché.

Reply:
I think that's a good idea!